nevergrad/optimization/optimizerlib.py

```diff
@@ -1425,32 +1425,126 @@ def _internal_tell_not_asked(self, candidate: p.Parameter, value: float) -> None
         raise base.TellNotAskedNotSupportedError
 
 
-class EMNA_TBPSA(TBPSA):
-    """Test-based population-size adaptation with EMNA.
+@registry.register
+class EMNA(base.Optimizer):
+    """Simple Estimation of Multivariate Normal Algorithm (EMNA).
     """
 
+    # pylint: disable=too-many-instance-attributes
+
+    def __init__(self, parametrization: IntOrParameter, budget: Optional[int] = None, num_workers: int = 1) -> None:
```

Comment:
Would it be possible to have additional inputs `naive`  and `anisotropic` here, and then use these options in the rest of the code?
The aim is to do something like the implementation of OnePlusOne here which is a good example of what we are aiming for (TBPSA is not a good example, I need to update it):
https://github.com/facebookresearch/nevergrad/blob/b8eaf6221ec8fc2985b7363ed39f7d2943a3fc2a/nevergrad/optimization/optimizerlib.py#L38-L160

That would avoid code duplication, and make it way easier to iterate upon on the long range.